### ISSUE TITLE
Support steam:// links

### ISSUE THREAD

**Issue by MateusRodCosta** [flathub/com.valvesoftware.Steam#925]
**Game information**

Not really, just the integration between OS and Steam flatpak

**Distribution name and version where applicable**
```
NAME="Fedora Linux"
VERSION="36.20220707.0 (Silverblue)"
```

**Flatpak info**

```
[mateusrc@centauro ~]$ flatpak --version
Flatpak 1.12.7
[mateusrc@centauro ~]$ flatpak --gl-drivers
nvidia-515-57
default
host
```

**Problem description**

It seem the Steam flatpak doesn't register as a handler for `steam://` links.
This prevents external programs from asking Steam to install or launch games.

**Does this issue reproduce with native Steam**

Have not tested

**Comment by nicokimmel** [comment]
This does work for me.
Arch, Gnome 42.3, also Flatpak version of Steam.
(It even lets me choose between Flatpak and native. Got both of them installed actually, since VR does not work with Flatpak currently. With only one installed it does not ask.)
![grafik](https://user-images.githubusercontent.com/31391271/178138039-2a7428e7-cd6d-44b0-822b-4db1d6083042.png)

**Comment by nicokimmel** [comment]
Actually, a thing thats not working for me is
`vrserver://quit` (IIRC) when closing SteamVR from the VR Headset itself.

**Comment by nanonyme** [comment]
Right, original poster is probably missing some xdg component that is necessary for this to work.

**Comment by 1player** [comment]
Not a Steam nor a Steam flatpak bug.

I'm running Silverblue as well, and opening custom URL handlers across Flatpaks is broken in some cases.

Opening `steam://run/306130` in Firefox flatpak doesn't do anything because IIRC Firefox registers an handler for all unknown schemes, breaking it.

But if you open a terminal and run `xdg-open steam://run/306130`, this Flatpak is started as intended.
